app/src/main/res/drawable/shortcut_note.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:id="@+id/shortcut_plus_background">
+        <shape android:shape="oval">
+            <solid android:color="@color/color_primary"/>
+        </shape>
+    </item>
+    <item android:bottom="@dimen/medium_margin" android:drawable="@drawable/ic_article_vector" android:left="@dimen/medium_margin" android:right="@dimen/medium_margin" android:top="@dimen/medium_margin" />
+</layer-list>
```

Comment:
why this new icon? Just reuse icon from the launcher

Reply:
1. I can't reuse it because it's too small. I'd need to do a new drawable with a bigger icon.
2. Even if I do it, it then looks like this:
![image](https://user-images.githubusercontent.com/85929121/227177059-0c501f14-af80-402a-af52-f53d0d491dca.png)
Wouldn't it be confusing for users?

Reply:
drawable/ic_launcher_monochrome.xml contains the foreground only as a .svg, you can use that. Showing the same icon twice is still less confusing than showing some other apps icon.

Reply:
I know, that's why I wrote that I need to create a new drawable with scaled icon:
![image](https://user-images.githubusercontent.com/85929121/227180061-388f63c9-91d0-4ad3-8d37-52818ff69a04.png)

Reply:
that icon I wrote above is a svg, you can resize it in any way you want. You can also use https://shapeshifter.design/ for converting the existing xml into an actual .svg file that you can resize or play around with.